data/committees.js

```diff
@@ -57,6 +58,19 @@ export default [
 		],
 		introImage: '/images/committees/studio/studio_motif_applied.png',
 		mission: 'Our mission is to teach skills revolving around video game development such as computer science, game design and art in order to help usher students into the game development industry.',
+		whatInternsWillDo:
```

Comment:
This comment applies for all the committees stuff - are you looking to have bullet points, etc.? If so, maybe this should be a JSX element.